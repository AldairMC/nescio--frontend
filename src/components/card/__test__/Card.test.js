import React from "react"
import renderer from "react-test-renderer"

import Card from "../Card"

describe("<Card></Card>", () => {
  test("Renders correctly", () => {
    const card = renderer.create(<Card></Card>).toJSON()
    expect(card).toMatchSnapshot()
  })
})
