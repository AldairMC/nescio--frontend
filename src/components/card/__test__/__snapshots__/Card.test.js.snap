// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card></Card> Renders correctly 1`] = `
<div
  className="card-container elevation"
/>
`;
